src/web/src/config.ecpp

```diff
@@ -484,14 +484,18 @@ UserInfo user;
             log_info_audit ("Request CREATE config SUCCESS");
             return HTTP_OK;
         }
-        catch (const BiosError &e) {
+    }
+    catch (const BiosError &e) {
+        if (request.isMethodPOST ()) {
             log_error_audit ("Request CREATE config FAILED");
-            http_die_idx (e.idx, e.what ());
         }
-        catch (const std::exception &e) {
+        http_die_idx (e.idx, e.what ());
+    }
+    catch (const cxxtools::SerializationError &e) {
+        if (request.isMethodPOST ()) {
             log_error_audit ("Request CREATE config FAILED");
-            http_die ("bad-request-document", e.what ());
         }
+        http_die ("bad-request-document", e.what ());
```

Comment:
If we throw the request-document message for non-POST requests, would it be correct?
Otherwise LGTM, the change is small and meaningful :)

Reply:
Since we don't deserialize the body on a GET request, we shouldn't be able to land here with a GET. Still, can't hurt to have it at this level.